### PR TITLE
Fix logging

### PR DESCRIPTION
**Pull request by OfryL**
## ℹ️ Change description:

- fix network logs for dev in locomotion app
- refactor server to use cloud logger
- format existing logs for better logs tracking

<sub>[templates: [bug](?expand=1&template=bug.md), [feature](?expand=1&template=feature.md)]</sub>

### CODE REVIEW

File: server/src/test/component_tests/api/v1/create-ride.test.ts [hunk 1]
```diff
@@ -19,7 +19,7 @@ const createUserAndLogin = async () => {
   const response = await request(app).post(`${baseUrl}/login/vert`)
     .send({ phoneNumber: '972501234567', code: '1234' })
     .set('x-loco-op-id', operationId);
-  console.log('response', response.body);
+  console.info('response', response.body);
```
Comment (OmerGery) [inline server/src/test/component_tests/api/v1/create-ride.test.ts:22]: logger?

**Review by OmerGery — APPROVED** [review]

**Review by avivbarniv — COMMENTED** [review]
Do you need all the console logs?

**Review by oriefrati — APPROVED** [review]

**Review by avivbarniv — APPROVED** [review]

**Comment by OfryL** [comment]
> Do you need all the console logs?

Yeah usefully for now

File: server/src/test/component_tests/api/v1/create-ride.test.ts (hunk 1 above)
Comment (OfryL) [reply server/src/test/component_tests/api/v1/create-ride.test.ts:22]: It's in test so it's cool

**Pull request merged**